roles/keycloak/templates/18.0.0/keycloak.conf.j2

```diff
@@ -32,3 +32,5 @@ log=file,console
 log-file={{ keycloak_logs_folder }}/server.log
 #log-level=INFO,org.infinispan:DEBUG,org.jgroups:DEBUG
 
+health-enabled=true
```

Comment:
I'd suggest to use `keycloak_health_enabled` config option to control this by chaning the line as follows:

```
health-enabled={{ keycloak_health_enabled | default(False) | string | lower }}
```